assets/js/src/simple-page-ordering.js

```diff
@@ -1,3 +1,6 @@
+import '../../css/scss/simple-page-ordering.scss';
+import 'jquery-ui-sortable';
```

Comment:
```suggestion
import 'jquery-ui-sortable';
import 'jquery';
```

jQuery is used in a number of places throughout. WordPress Scripts has it set as a dependency so no changes needed in the webpack config,